app/services/SpringBootVersionService.scala

```diff
@@ -18,17 +19,32 @@ import scala.xml.XML
   */
 object SpringBootVersionService {
 
-  private val propertyRegex = """\$\{(.*)\}""".r
-  private val baseSpringbootPomUrl = "https://raw.githubusercontent.com/spring-projects/spring-boot/v"
+  private val propertyRegex = """\$\{(.*)}""".r
+  private val libraryVersion = """\s+library\("[^"]+", "([^"]+)".*""".r
+  private val groupName = """\s+group\("([^"]+)".*""".r
+  private val artefactName = """\s+"([^"]+)".*""".r
+  private val endBlock = """\t}""".r
+  private val innerEndBlock = """\t\t}""".r
+
+  private val baseSpringbootUrl = "https://raw.githubusercontent.com/spring-projects/spring-boot/v"
   private val pomPath = "/spring-boot-dependencies/pom.xml"
+  private val gradlePath = "/spring-boot-dependencies/build.gradle"
+  private val mavenVersions = List("2.0", "2.1", "2.2")
+  private val gradleVersions = List("2.3", "2.4", "2.5", "2.6", "2.7")
```

Comment:
Not sure about the naming here. These look like spring boot versions.

Reply:
I think he wanted to mean that starting Spring Boot 2.3 the POM switch from Maven to Gradle. 

However do we really want to fix versions for this ? Do you want to add explicit support for each Spring boot versions ?

Reply:
@clemstoquart hello, yep the meaning was springboot version using maven or gradle. I renamed it springboot2VersionWithPom
@celcius112 springboot 2.3+ is the new default as the structure of build.gradle in springboot seems unchanged.